docs/docs/08-working-with-the-browser.md

```diff
@@ -55,6 +55,6 @@ _Mounted_ composite components also support the following method:
 
 ## Browser Support
 
-React supports most popular browsers, including Internet Explorer 9 and above.
+React supports most popular browsers, including Internet Explorer 9 and above. After [React v15.0](https://facebook.github.io/react/blog/2016/04/07/react-v15.html), we've decided [not to actively support Internet Explorer 8](https://facebook.github.io/react/blog/2016/01/12/discontinuing-ie8-support.html) with any new fixes, as it is running fine in its current form.
```

Comment:
Agree. Especially using "running". Saw the blog explaining the situation on IE was using form, so used the same definition. Should I rephrase / You rephrase and republish or should we just leave it? It would be a nice addition to the browser support section as this is exactly where this kind of information should be rather just in a blog post. Perhaps just cut of "running fine" and say we do not actively support?